Documentation/platforms/sim/sim/index.rst

```diff
@@ -0,0 +1,56 @@
+===
+SIM
+===
+
+It is possible to run NuttX in a simulator called ``sim``, but currently it only
+supports Linux as host platform.
```

Comment:
Is this (still) true? I think it was used on both macOS and Windows.

Reply:
It was developed for Linux, all main features depends on Linux subsystems: BT, I2C, SPI, etc

Reply:
But, other feature(especially network and hostfs) work on macOS.

Reply:
Ok, fixed!